_posts/2017-08-25-introducing-sandboxfs.md

```diff
@@ -99,15 +101,20 @@ available as an open-source project under the Bazel umbrella! See:
 
 > https://github.com/bazelbuild/sandboxfs/
 
-Keep in mind that this project is still an experiment and highly subject to change. In
-particular, be aware that the command line and the data formats are most
-certainly going to mutate (for simplicity if anything). But the
+Keep in mind that this project is still an experiment and highly subject to
+change. In particular, be aware that the command line and the data formats
+are most certainly going to mutate (for simplicity if anything). But the
 current code is now sufficient to experiment and play with.
 
+As sandboxfs is now effectively an open-source project, please take a look
```

Comment:
remove "effectively" unless it's not really open source

Reply:
Done.